runlarky/src/main/proto/larky.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package com.verygood.security.mode.grpc;
+option java_outer_classname = "LarkyProto";
+option java_multiple_files = true;
+
+message LarkyProcessRequest {
```

Comment:
@moehajj plz review this

Reply:
@aslepakurov I think we should use the `request` & `response` objects here, instead of `bytes`, for the generic proxy flows. It would be best to wait for the PR that introduces the comprehensive versions of these objects, and this way you can avoid a breaking changes that's expected very soon!

Reply:
Ok, I will make just a working grpc version without merging, just to build other changes on this. As soon as you would have proto, please, notify, so that I can make appropriate changes